_docs/tooltips-lab.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: docs
+title: Tooltips Lab
+---
+
+**Tooltips Lab** can be used to quickly create callouts and triggers that allow you to present extra information on your presentation slides with a click of a button. Tooltips Lab allows you to create a new tooltip from scratch, or to use existing shapes on your presentation slides and convert them into a tooltip.
+
+## <a class="anchor-bookmark" id="create-tooltip"></a> Create Tooltip
+**Create Tooltip** allows you to create a complete tooltip from scratch, with a single click of a button. The tooltip created consists of a callout shape and a trigger shape. In slide-show mode, the hidden callout shape will be revealed on click of the trigger shape, and it can be hidden again on a subsequent click.
+
+To use this feature, go to `Tooltips Lab` and select `Create Tooltip`.
+
+<p>
+  <img class="box-shadow" src="{{ site.baseurl }}/img/docs/tooltips-lab/create-tooltip-feature.png">
+</p>
+
+You can now go into slideshow mode to try the tooltip!
+
+<p>
+  <img class="box-shadow" src="{{ site.baseurl }}/img/docs/tooltips-lab/create-tooltip.gif" width="450">
+</p>
+
+> **Note:** Clicking "Create Tooltip" will generate a blank callout and default trigger shape for you. You can then add in your own text and information, or to format the shapes as desired after it is generated!
+
+> **Tip:** If you are adding pictures or additional shapes to the callout, you may want to group them together to make moving them around easier! <p><img class="box-shadow" src="{{ site.baseurl }}/img/docs/tooltips-lab/create-tooltip-group-tip.png" width="450"></p>
```

Comment:
I thought this tip is incorrect? Because it will destroy the associated animation.